components/list/list-item-generic-layout.js

```diff
@@ -143,8 +154,8 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 			<slot name="control-action" class="d2l-cell" data-cell-num="3"></slot>
 			<slot name="control" class="d2l-cell" data-cell-num="4"></slot>
 			<slot name="actions" class="d2l-cell" data-cell-num="6"></slot>
-
 			<slot name="content" @focus="${this._preventFocus}"></slot>
+			<slot name="nested"></slot>
```

Comment:
This introduces a new slot for the "nested" content for the generic layout and list-item.  This enables the list-item to control the layout of the nested content in terms of the CSS grid.  For instance, it enables the nested content to span underneath the optional secondary actions on the right side - this would not be possible if the nested list was distributed to the default slot.